src/Microsoft.ML.LightGBM/doc.xml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+    <!--  
+    The following text describes the LightGbm algorithm details.
+    It's used for the remarks section of all LightGbm-based trainers (binary, multiclass, regression, ranking)
+    -->
+    <member name="LightGBM_remarks">
+      <remarks>
+        LightGBM is an open source implementation of boosted trees. For implementation details, please see 
```

Comment:
>boosted trees [](start = 53, length = 13)

gradient boosted decision trees. (or Gradient Boosting Machine for GBM?)

Reply:
changed to "Gradient Boosting Decision Tree" based on Tom's paper title.

---
In reply to: [263944693](https://github.com/dotnet/machinelearning/pull/2886#discussion_r263944693) [](ancestors = 263944693)